test/example-app/index.js

```diff
@@ -737,6 +755,9 @@ const App = {
     [MovieSearch.key]: MovieSearch
   },
   hydrators: {
+    legacyMethodHydrator: {
+      source: "return z.legacyScripting.run(bundle, 'hydrate.method');"
```

Comment:
Glad to see these stay nice and short, with the complexity inside legacyScripting Runner.